### ISSUE TITLE
Implement edit mode in redesigned account switcher

### ISSUE THREAD

**Issue by slaweet** [LiskHQ/lisk-desktop#44]
### Expected behaviour
The edit mode should allow to: 
- Remove account with a confirmation step
![04_favorite_ids- -favorite-ids-multi-remove-dashboard-lisk-xl](https://user-images.githubusercontent.com/1254342/33365793-a173c258-d4e9-11e7-9087-6d557be882b1.png)

The drag and drop of accounts will be done in another ticket https://github.com/LiskHQ/lisk-wallet/issues/52
### Actual behaviour
After #9 the account switcher will not have the edit mode and remove will have no confirmation step.
